Core/include/Acts/EventData/ParticleHypothesis.hpp

```diff
@@ -44,10 +44,13 @@ class SinglyChargedParticleHypothesis
   static SinglyChargedParticleHypothesis electron() {
     return SinglyChargedParticleHypothesis(PdgParticle::eElectron);
   }
+  static SinglyChargedParticleHypothesis kaon() {
+    return SinglyChargedParticleHypothesis(PdgParticle::eKaon);
+  }
   static SinglyChargedParticleHypothesis proton() {
     return SinglyChargedParticleHypothesis(PdgParticle::eProton);
   }
-
+  
```

Comment:
```suggestion

```

Reply:
Oh, I understood that you wanted to keep the separation line so I returned it. In the current code it's there.

Reply:
Right I think the diff is broken. I meant to remove the whitespace. The format CI job will complain about this